core/whistle_apps-1.0.0/src/whapps_controller.erl

```diff
@@ -59,13 +59,12 @@ start_app(App) ->
 stop_app(App) when not is_atom(App) ->
     stop_app(wh_util:to_atom(App));
 stop_app(App) ->
-    _ = application:stop(App),
-    lager:info("stopped kazoo application ~s", [App]).
+    case application:stop(App) of
+      ok -> lager:info("stopped kazoo application ~s", [App]), ok;
+      Err -> Err
+    end.
 
--spec restart_app(atom() | nonempty_string() | ne_binary()) ->
-                         {'ok', pid() | 'undefined'} |
-                         {'ok', pid() | 'undefined', term()} |
-                         {'error', term()}.
+-spec restart_app(atom() | nonempty_string() | ne_binary()) -> 'ok' | {'error', term()}.
```

Comment:
@lazedo I don't follow.
This function calls `application:start(atom())` which returns `'ok' | {'error', term()}.`
As per documentation: http://www.erlang.org/doc/apps/kernel/application.html#start-1